### ISSUE TITLE
georeferencing off for a batch of Bell Musuem Sceloporus occidentalis records

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3310]
**georeferencing off for a batch of Bell Musuem Sceloporus occidentalis records**

The georeferencing on these Sceloporus occidentalis records from the Bell Museum seems suspect. The locality data for the ones I clicked through on all indicate California.

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/8e9ec9dadf2a1e4e77877c140a1cb1f1:028e86f7477118af5421bf798f0687abd07bab7155dfd5ef99c83b35f4892a913be3bab4def57d426b2f3334ab86e521eac0f18cd821759900c4f4311f04d1ff)
System: Firefox 86.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/occurrence/search?geometry=POLYGON((-94.59334%2043.49231,-89.28145%2043.49231,-89.28145%2045.61817,-94.59334%2045.61817,-94.59334%2043.49231))&has_coordinate=true&has_geospatial_issue=false&occurrence_status=present&taxon_key=2451234
Window size: width 1366 - height 665
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-03-09T18:33:57.773Z',to:'2021-03-09T18:39:57.773Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-03-09T18:33:57.773Z',to:'2021-03-09T18:39:57.773Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: WARNING

**Comment by smsfrn** [comment]
I logged this on the site and since found a number of additional records with a similar issue (see below for list). All these have a mismatch between coordinates and stateprovince field.

I've emailed a contact at the data provider with this list.

Records with issues (gbifids):  "2266290754" "2266290049" "2266290524" "2266291473" "2266289517" "2266290218" "2266290345" "2266290390" "2266290704" "2266289041" "2266288685" "2266288706" "2266288993" "2266289183" "2266288900" "2266290015" "2266288342" "2266289027" "2266288713" "2266288647" "2266289708" "2266289898" "2266288598" "2266288418" "2266289327" "2266289442" "2266289608" "2266287589" "2266287782" "2266288042" "2266288193" "2266288630" "2266289015" "2266286887" "2266287427" "2266287877" "2266287880" "2266287971" "2266288046" "2266288216" "2266288359" "2266288407" "2266288442" "2266288777" "2266289139" "2266288734" "2266288250" "2266287728" "2266287825"
